docs/proto/index.html

```diff
@@ -362,6 +362,14 @@ <h2>Table of Contents</h2>
             <a href="#x%2fcash%2fcodec.proto">x/cash/codec.proto</a>
             <ul>
               
+                <li>
```

Comment:
Why is this in the diff???
This came from another PR, right?

Reply:
Somebody forgot to generate the docs moist likely. I have committed everything produced by `make protoc`